src/transformers/models/beit/configuration_beit.py

```diff
@@ -92,6 +92,8 @@ class BeitConfig(PretrainedConfig):
             Number of convolutional layers to use in the auxiliary head.
         auxiliary_concat_input (:obj:`bool`, `optional`, defaults to :obj:`False`):
             Whether to concatenate the output of the auxiliary head with the input before the classification layer.
+        semantic_ignore_index (:obj:`int`, `optional`, defaults to 255):
```

Comment:
```suggestion
        loss_ignore_index (:obj:`int`, `optional`, defaults to 255):
```
Or `semantic_loss_ignore_index` if the semantic is crucial, but we can't get rid of the loss.